src/main/webapp/app/pages/newsPage/ChangedAnnotatonListItem.tsx

```diff
@@ -58,6 +64,32 @@ export const ChangedAnnotationListItem = (props: {
   }
 
   const geneColumnIndex = getColumnIndexByName(annotationColumnHeader, GENE);
+  const mutationColumnIndex = getColumnIndexByName(
+    annotationColumnHeader,
+    MUTATION
+  );
+
+  if (mutationColumnIndex > -1 && geneColumnIndex > -1) {
+    // transform the gene and mutation input to a link, ignore the inputs with comma, pipe or slash
+    props.data.forEach(row => {
+      const geneInput = row.content[geneColumnIndex].content;
+      const mutationInput = row.content[mutationColumnIndex].content;
+      if (
+        typeof geneInput === 'string' &&
+        typeof mutationInput === 'string' &&
+        geneInput !== 'ESR1'
```

Comment:
This is not a good condition you want to use.

Reply:
I removed the `geneInput !== 'ESR1'` and placed in `linkableMutationName `